### PR TITLE
added integration test to check if containers in app updates are correctly processed

### PR DESCRIPTION
**Pull request by unterstein**
added integration test to check if containers in app updates are correctly processed

Related to https://github.com/mesosphere/marathon/pull/4228 which also fixes the behavior, but fix was already in master.

### CODE REVIEW

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

File: src/test/scala/mesosphere/marathon/api/v2/json/AppDefinitionTest.scala [hunk 1]
```diff
@@ -924,4 +924,15 @@ class AppDefinitionTest extends MarathonSpec with Matchers {
     roundTripped should equal(appDef)
     roundTripped.container.map(_.portMappings) should equal(appDef.container.map(_.portMappings))
   }
+
+  test("container change in AppUpdate should be stored") {
```
Comment (meichstedt) [inline src/test/scala/mesosphere/marathon/api/v2/json/AppDefinitionTest.scala]: shouldn't this test case be in `AppUpdateTest`?

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

**Comment by unterstein** [comment]
retest this please

**Comment by meichstedt** [comment]
👍

**Pull request merged**